pkg/kube/ready.go

```diff
@@ -230,18 +226,18 @@ func (c *ReadyChecker) isPodReady(pod *corev1.Pod) bool {
 			return true
 		}
 	}
-	c.log("Pod is not ready: %s/%s", pod.GetNamespace(), pod.GetName())
```

Comment:
I choose a capital letter for the first letter because it represent the resource.